Exam/CC1/src/exercice2/SortedContainer.java

```diff
@@ -0,0 +1,61 @@
+package exercice2;
+
+public class SortedContainer implements ISortable {
+
+	int[] elements = new int[15];
+	int tailleTableau =0;
+
+
+	@Override
+	public void sort(int[] list, int listSize) {
+		int longueur = listSize;
+		int tampon = 0;
+		boolean permut;
+
+		do {
```

Comment:
Hmmm... This works but unfortunately is totally inefficient! ;).
You sort more than one items already sorted...